quizbizz/src/pages/quizanswers/quizquestionanswers/QuizQuestionAnswer.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+
+const QuizQuestionAnswer = ({ question, correctAnswer, answerSelected }) => {
+  const options = question?.options;
+  return (
+    <div className="quiz-question-answer-container">
+      <h2 className="primary-text-color font-medium-large weight-bold">
+        {`${question?.questionNumber}. ${question?.question}`}
+      </h2>
+      <div className="spacer-2"></div>
+      <ol className="quiz-questions-container list-style-upper-alphabets">
+        <li className="font-medium-large primary-text-color">{options?.[0]}</li>
+        <div className="spacer-1"></div>
+        <li className="font-medium-large primary-text-color">{options?.[1]}</li>
+        <div className="spacer-1"></div>
+        <li className="font-medium-large primary-text-color">{options?.[2]}</li>
+        <div className="spacer-1"></div>
+        <li className="font-medium-large primary-text-color">{options?.[3]}</li>
+      </ol>
```

Comment:
Suggestion: here you can map over options array, instead of manually tying all items

Reply:
If i do that there comes margin issue that is why i am doing like this

Reply:
but use map for clean code , & fix CSS manually ,